components/raftstore/src/coprocessor/split_check/size.rs

```diff
@@ -224,7 +224,7 @@ pub fn get_region_approximate_size(
 }
 
 /// Get region approximate split keys based on default, write and lock cf.
-fn get_approximate_split_keys(
+pub fn get_approximate_split_keys(
```

Comment:
it's based on size, shouldn't use it.

Reply:
> it's based on size, shouldn't use it.
to approximately split region by keys, there's no way to use key count.